docs/source/caching/normalized-cache.mdx

```diff
@@ -33,68 +33,53 @@ query AuthorById($id: String!) {
+
+## Memory cache
 
-## Storing your data in memory
+Apollo Android comes with a `MemoryCache` that will store your data in memory. To use it, add the `apollo-normalized-cache` artifact to your dependencies in your `build.gradle[.kts]` file:
 
-Apollo Android comes with a `MemoryCache` that will store your data in memory:
+```kotlin:title=build.gradle[.kts]
+dependencies {
+  implementation("com.apollographql.apollo3:apollo-normalized-cache:x.y.z")
+}
+```
 
-<MultiCodeBlock>
+And add it to your `ApolloClient`:
 
 ```kotlin
 // Create a 10MB MemoryCacheFactory
 val cacheFactory = MemoryCacheFactory(maxSizeBytes = 10 * 1024 * 1024)
-
 // Build the ApolloClient
 val apolloClient = ApolloClient.Builder()
   .serverUrl("https://...")
+  // normalizedCache() is an extension function on ApolloClient.Builder
   .normalizedCache(cacheFactory)
   .build()
 ```
 
-</MultiCodeBlock>
+Because the normalized cache is optional, `normalizedCache()` is an extension function on `ApolloClient.Builder()` that's defined in the `apollo-normalized-cache` artifact. It takes a `NormalizedCacheFactory` as a parameter so that it can create the cache in the background if needed.
 
-## Persisting your data in a SQLite database
+A `MemoryCache` is a least recently used (LRU) cache. It keeps entries in memory based on two conditions:
 
-If the amount of data you store becomes too big to fit in memory or if you want your data to persist between app restarts, you can also use a `SqlNormalizedCacheFactory`. A `SqlNormalizedCacheFactory` will store your data in a [SQLDelight](https://github.com/cashapp/sqldelight) database and is defined in a separate dependency:
+* `maxSizeBytes`: the maximum size in bytes the cache may occupy
+* `expireAfterMillis`: after what timeout each entry in the cache treated as expired. By default there is no timeout.
 
-```kotlin:title=build.gradle.kts
-dependencies {
-  implementation("com.apollographql.apollo3:apollo-normalized-cache-sqlite:x.y.z")
-}
-```
+When your app is stopped, data in the `MemoryCache` is lost forever. If you need to persist data, you can use the SQLite cache.
 
-**Note:** The `apollo-normalized-cache-sqlite` dependency has Kotlin multiplatform support and has multiple variants (`-jvm`, `-android`, `-ios-arm64`,...). If you are targeting Android and using custom `buildTypes`, you will need to help Gradle resolve the correct artifact by defining [matchingFallbacks](https://developer.android.com/studio/build/dependencies?authuser=2#resolve_matching_errors):
+## SQLite cache
 
-<MultiCodeBlock>
+The SQLite cache uses [SQLDelight](https://github.com/cashapp/sqldelight) to store data persistently if you need to to persist data accross app restarts or simply if the data becomes too big to fit in memory.
```

Comment:
Nit
```suggestion
The SQLite cache uses [SQLDelight](https://github.com/cashapp/sqldelight) to store data persistently. It can be used when you need to persist data across app restarts or simply if the data becomes too big to fit in memory.
```